[2024-02-28T12:10:26.899+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gcs_snowflake_etl.gcs_object_exists manual__2024-02-28T12:08:15.242514+00:00 [queued]>
[2024-02-28T12:10:26.915+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gcs_snowflake_etl.gcs_object_exists manual__2024-02-28T12:08:15.242514+00:00 [queued]>
[2024-02-28T12:10:26.917+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 3
[2024-02-28T12:10:26.944+0000] {taskinstance.py:2191} INFO - Executing <Task(GCSObjectExistenceSensor): gcs_object_exists> on 2024-02-28 12:08:15.242514+00:00
[2024-02-28T12:10:26.955+0000] {standard_task_runner.py:60} INFO - Started process 111 to run task
[2024-02-28T12:10:26.960+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'gcs_snowflake_etl', 'gcs_object_exists', 'manual__2024-02-28T12:08:15.242514+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/GCS_to_Snowflake_DAG.py', '--cfg-path', '/tmp/tmpne2r81ok']
[2024-02-28T12:10:26.965+0000] {standard_task_runner.py:88} INFO - Job 5: Subtask gcs_object_exists
[2024-02-28T12:10:27.048+0000] {task_command.py:423} INFO - Running <TaskInstance: gcs_snowflake_etl.gcs_object_exists manual__2024-02-28T12:08:15.242514+00:00 [running]> on host b9a0bf70b495
[2024-02-28T12:10:27.244+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='stellar' AIRFLOW_CTX_DAG_ID='gcs_snowflake_etl' AIRFLOW_CTX_TASK_ID='gcs_object_exists' AIRFLOW_CTX_EXECUTION_DATE='2024-02-28T12:08:15.242514+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-28T12:08:15.242514+00:00'
[2024-02-28T12:10:27.247+0000] {gcs.py:98} INFO - Sensor checks existence of : us_stock_market, US-Stock-Market-Dataset.csv
[2024-02-28T12:10:27.271+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-02-28T12:10:27.283+0000] {base.py:83} INFO - Using connection ID 'gcp_conn_id' for task execution.
[2024-02-28T12:10:27.973+0000] {base.py:295} INFO - Success criteria met. Exiting.
[2024-02-28T12:10:27.988+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=gcs_snowflake_etl, task_id=gcs_object_exists, execution_date=20240228T120815, start_date=20240228T121026, end_date=20240228T121027
[2024-02-28T12:10:28.059+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-28T12:10:28.138+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
